packages/core/src/controllers/ApiController.ts

```diff
@@ -184,14 +191,24 @@ export const ApiController = {
   },
 
   prefetch() {
-    state.prefetchPromise = Promise.race([
-      Promise.allSettled([
-        ApiController.fetchFeaturedWallets(),
-        ApiController.fetchRecommendedWallets(),
-        ApiController.fetchNetworkImages(),
-        ApiController.fetchConnectorImages()
-      ]),
-      CoreHelperUtil.wait(3000)
-    ])
+    const promises = [
+      ApiController.fetchFeaturedWallets(),
+      ApiController.fetchRecommendedWallets(),
+      ApiController.fetchNetworkImages(),
+      ApiController.fetchConnectorImages()
+    ]
+    if (OptionsController.state.enableAnalytics === undefined) {
```

Comment:
is it necessary to check agains `undefined` directly?

Reply:
Yes because if the param is provided through the code (could be `true` or `false`) it takes precedence over the Cloud setting.